lib/lib_rss.php

```diff
@@ -189,7 +189,14 @@ function html_only_entity_decode($text) {
 }
 
 function prepareSyslog() {
-	return COPY_SYSLOG_TO_STDERR ? openlog("FreshRSS", LOG_PERROR | LOG_PID, LOG_USER) : false;
+	if (defined('SYSLOG_PREPARED')) {
+		return null;
+	}
+	define('SYSLOG_PREPARED', true);
```

Comment:
Why do you use a constant? I think we should remove other calls to `prepareSyslog` and keep only the one you added in this PR (not sure if it's the best place for that btw, but it's nitpicking). This way we can remove the use of `SYSLOG_PREPARED`

Reply:
Yes indeed. It was a performance consideration to avoid the cost of `openlog()` when it is not used (i.e. most of the time).

Reply:
Done https://github.com/FreshRSS/FreshRSS/pull/2745/commits/4b93ced3039031dc0d4c7c74118970accca2b15e

Reply:
P.S. Not that bad thanks to
> 

LOG_ODELAY | (default) delay opening the connection until the first             message is logged
-- | --